docs/mesos/PrepareExternalServices.md

```diff
@@ -0,0 +1,54 @@
+# Prepare External Services for Mesos + Calico Deployment
+*Most Mesos clusters will run these services outside of their core Mesos cluster, separate from the Masters and Agents which read from them.*
+
+## Docker
+
+- **Docker must be installed on each Mesos Agent** that is deploying calico via the packaged Calico container (recommended). Users who prefer to [run Calico as a baremetal service](#) do not need to install Docker on each Agent.
+- **Docker must be installed on Mesos Master** if etcd and ZooKeeper are being deployed as Docker containers. Users who are running etcd / ZooKeeper elsewhere do not need to install Docker on each Master.
+
+Run the following commands to install Docker:
```

Comment:
I've thought about it a bit, and I think we should leave this in as is, and work towards removing it later on